### PR TITLE
[FEAT] - adapted styles for the mobile

### PR DESCRIPTION
**Pull request by vlad-lisnenko**

### CODE REVIEW

File: styles/course_style.css [hunk 1]
```diff
@@ -86,6 +86,8 @@
     position: relative;
 }
 
+
+/*1*/
```
Comment (scratchw0w) [inline styles/course_style.css:90]: what does 1 mean?

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: scripts/course_slider_script.js [hunk 2]
```diff
@@ -111,3 +112,22 @@ PREV_BUTTON.addEventListener(EVENT_CLICK, () => {
...
+    touchStartX = e.touches[0].clientX;
+});
+
+ED_SLIDER.addEventListener("touchmove", (e) => {
+    touchEndX = e.touches[0].clientX;
+});
+
+ED_SLIDER.addEventListener("touchend", () => {
+    if (touchStartX - touchEndX > SIZE_PX_FOR_TOUCH) {
+        changeSlide(1);
```
Comment (scratchw0w) [inline scripts/course_slider_script.js:126]: not 1 and -1, use constants like we've done before

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: styles/course_style.css (hunk 1 above)
Comment (vlad-lisnenko) [reply styles/course_style.css:90]: its was flag  for  me

File: scripts/course_slider_script.js (hunk 2 above)
Comment (vlad-lisnenko) [reply scripts/course_slider_script.js:126]: forgot

File: styles/course_style.css (hunk 1 above)
Comment (scratchw0w) [reply styles/course_style.css:90]: remove it please

Comment (vlad-lisnenko) [reply styles/course_style.css:90]: ok bro

**Review by scratchw0w — APPROVED** [review]
Good Job!

**Pull request merged**